src/dispatch/participant_activity/service.py

```diff
@@ -66,6 +70,16 @@ def get_last_participant_activity(
     )
 
 
+def get_all_case_participant_activities_for_case(
+    db_session: SessionLocal,
```

Comment:
```suggestion
    db_session: Session,
```

`SessionLocal` is just `Any` AFAIK.